web/ui/react-app/src/pages/serviceDiscovery/Services.tsx

```diff
@@ -20,100 +19,95 @@ export interface TargetLabels {
   isDropped: boolean;
 }
 
-const Services: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix }) => {
-  const { response, error } = useFetch<ServiceMap>(`${pathPrefix}/api/v1/targets`);
-
-  const processSummary = (response: ServiceMap) => {
-    const targets: any = {};
+export const processSummary = (activeTargets: Target[], droppedTargets: DroppedTarget[]) => {
```

Comment:
Do `processSummary` and `processTargets` need to be exported? Seems they're only used from within this file.

Reply:
Yes, for testing.

Reply:
I don't see them being used in any tests yet though... but it would make sense at that point, yeah.

Reply:
yes, yes...exported for the future tests :)